src/colorspace.rs

```diff
@@ -3,6 +3,8 @@ use crate::Color;
 
 pub trait ColorSpace {
     fn from_color(c: &Color) -> Self;
+    // NOTE: Clippy's suggestion might be correct here
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
so what happens if we change it to `self`?

Reply:
Seems to not require much work, I didn't even try it xD